jobserver/releases.py

```diff
@@ -7,8 +7,29 @@
 from .models.outputs import absolute_file_path
 
 
+class ReleaseFileAlreadyExists(Exception):
+    pass
```

Comment:
This makes the API code really readable 👍👍

Reply:
yeah internal logic exceptions are a Good Idea, I think